Projeto-Stream/src/views/Home/homeDetail.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <main
+    :style="{
+      backgroundImage: `linear-gradient(to right, black, rgba(0, 0, 0, 0.4)), url(${backgroundUrl})`
+    }"
+    class="bg-fixed bg-center bg-cover h-screen py-24 px-10 grid items-center">
+    <section>
+      <div class="p-5 w-full md:w-4/12">
```

Comment:
A suggestion would be to place this div class inside the section and delete this div